tests/unit/remote/ssh/test_connection.py

```diff
@@ -64,3 +76,28 @@ def test_walk(tmp_path, ssh):
             paths.add(posixpath.join(root, entry))
 
     assert paths == expected
+
+
+@pytest.mark.skipif(
+    platform.system() != "Darwin", reason="Only works on OSX (because of APFS)"
+)
+def test_reflink(chdir_tmp, foo, ssh):
```

Comment:
you could simply use `repo_dir` instead of `chdir_tmp, foo`

Reply:
yes, but I don't need all the repository stuff :sweat_smile: I'd prefer to leave it like it is

Reply:
@mroutis but you are creating new fixtures for no reason, it is better to reuse the already existing ones. What if you would need `bar`, you would create yet another fixture? Sounds bad.

Reply:
Or, I guess, if we would to go that way, we would probably make repo_dir use foo fixture, but that is quite a hustle :(

Reply:
well, it's not for no reason, it saves 12 lines :sweat_smile: 

I try to stay away from `repo_dir` in unit tests, since the requirements to write a tests should be minimal; if I change `foo` for `repo_dir` I'd need to remove `repo_dir.BAR` every time (or use another name, like `repo_dir / link`)